content/en/contribute/code/core/build-commands.md

```diff
@@ -46,7 +45,9 @@ For developers to execute to test `cht-core`.
 | `unit-shared-lib`             | Executes the unit test suite on all `shared-lib` modules. |
 | `unit-webapp`                 | Executes the unit test suite on `webapp`. |
 | `unit-webapp-continuous`      | Executes the unit test suite on `webapp`, and re-runs on code change. |
+| `wdio-default-mobile-local`   | Compiles the app and executes the mobile e2e test suite. |
```

Comment:
There's a new job now to test cht-form with wdio. Should that also be documented?

Reply:
There's also "build-cht-form"... I'll get Josh to document these.